packages/postgres-database/src/simcore_postgres_database/utils_users.py

```diff
@@ -18,17 +21,17 @@ class UserNotFoundInRepoError(BaseUserRepoError):
 
 class UsersRepo:
     @staticmethod
-    async def get_role(conn: SAConnection, user_id: int) -> UserRole:
+    async def get_role(conn: SAConnection, user_id: int) -> str:
         if value := await conn.scalar(
             sa.select(users.c.role).where(users.c.id == user_id)
         ):
-            return UserRole(value)
-        raise UserNotFoundInRepoError()
+            return cast(str, value)
+        raise UserNotFoundInRepoError
 
     @staticmethod
     async def get_email(conn: SAConnection, user_id: int) -> str:
         if value := await conn.scalar(
             sa.select(users.c.email).where(users.c.id == user_id)
         ):
-            return f"{value}"
-        raise UserNotFoundInRepoError()
+            return cast(str, value)
```

Comment:
same here. email is already a string... why cast it? this will make pylance/mypy fail to find issues later on